src/renderer/features/shared/components/library-header.module.scss

```diff
@@ -133,6 +135,7 @@
     width: 100%;
     height: 100%;
     background: var(--bg-header-overlay);
+    opacity: 0.6;
```

Comment:
Adding `opacity: 0.6` conflicts with the original `var(--bg-header-overlay)` when the background image is disabled. Either remove the opacity css value or make it conditional based on album image being enabled.